index.js

```diff
@@ -122,7 +122,7 @@ class TestRail {
 			output.log(`The response is ${JSON.stringify(res.data)}`);
 			return res.data;
 		}).catch(error => {
-			output.error(`Cannot add result for case due to ${error}`);
+			output.error(`Cannot add result for case due to ${error.response.data.error}`);
```

Comment:
might want to consider being a bit defensive here just incase error.response.data is undefined or error.response.data.error is undefined too

Reply:
sorted, cheers